client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx

```diff
@@ -1,21 +1,57 @@
-import { Heading, Link, Box, HStack } from '@chakra-ui/layout';
+import { Heading, Link, Box, HStack, Text } from '@chakra-ui/layout';
+import { useConfirmDelete } from 'chakra-confirm';
+
 import { LinkButton } from 'chakra-next-link';
 import { NextPage } from 'next';
+import { useRouter } from 'next/router';
 import React from 'react';
+import { Button } from '@chakra-ui/react';
 import { Card } from '../../../../components/Card';
 import ProgressCardContent from '../../../../components/ProgressCardContent';
-import { useChapterQuery } from '../../../../generated/graphql';
+import {
+  useChapterQuery,
+  useDeleteChapterMutation,
+} from '../../../../generated/graphql';
 import { useParam } from '../../../../hooks/useParam';
 import styles from '../../../../styles/Page.module.css';
 import { Layout } from '../../shared/components/Layout';
+import { CHAPTERS } from '../../../chapters/graphql/queries';
+import { VENUE } from '../../Venues/graphql/queries';
```

Comment:
```suggestion
import { VENUES } from '../../Venues/graphql/queries';
```
The `VENUE` query just fetches a single venue, but we need to update the collection of venues.